autopilot/betweenness_centrality_test.go

```diff
@@ -87,48 +85,52 @@ func TestBetweennessCentralityWithNonEmptyGraph(t *testing.T) {
 		},
 	}
 
+	workers := []int{-1, 1, 3, 9, 100}
```

Comment:
Just return an error in case of non-valid num workers?

Reply:
done